gui/src/main/java/de/azubi/kniffel/gui/fx/controllerhandling/GameController.java

```diff
@@ -0,0 +1,500 @@
+            for (int player1 = 0; player1 < MAX_PLAYERS; player1++) {
+                String textRealScore = scores[player1][i] == -1 ? "" : Integer.toString(scores[player1][i]);
+                scoreText[player1] = textRealScore;
+            }
+            tableView.getItems().add(new ScoreRow(ScoreUtils.lower(i + 1), scoreText));
+        }
+
+        for (int i = 0; i < MAX_SCORE_NAME; i++) {
+            String[] scoreText = new String[MAX_PLAYERS];
+            for (int player2 = 0; player2 < MAX_PLAYERS; player2++) {
+                String textRealScore = scores[player2][i] == -1 ? "" : Integer.toString(scores[player2][i]);
+                scoreText[player2] = textRealScore;
+            }
+            tableView.getItems().add(new ScoreRow(ScoreUtils.lower(i + 2), scoreText));
+        }
+
+        for (int i = 0; i < MAX_SCORE_NAME; i++) {
+            String[] scoreText = new String[MAX_PLAYERS];
+            for (int player3 = 0; player3 < MAX_PLAYERS; player3++) {
+                String textRealScore = scores[player3][i] == -1 ? "" : Integer.toString(scores[player3][i]);
+                scoreText[player3] = textRealScore;
+            }
+            tableView.getItems().add(new ScoreRow(ScoreUtils.lower(i + 3), scoreText));
+        }
+
+        for (int i = 0; i < MAX_SCORE_NAME; i++) {
+            String[] scoreText = new String[MAX_PLAYERS];
+            for (int player4 = 0; player4 < MAX_PLAYERS; player4++) {
+                String textRealScore = scores[player4][i] == -1 ? "" : Integer.toString(scores[player4][i]);
+                scoreText[player4] = textRealScore;
+            }
+            tableView.getItems().add(new ScoreRow(ScoreUtils.lower(i + 4), scoreText));
+        }
+
+        for (int i = 0; i < MAX_SCORE_NAME; i++) {
+            String[] scoreText = new String[MAX_PLAYERS];
+            for (int player5 = 0; player5 < MAX_PLAYERS; player5++) {
+                String textRealScore = scores[player5][i] == -1 ? "" : Integer.toString(scores[player5][i]);
+                scoreText[player5] = textRealScore;
+            }
+            tableView.getItems().add(new ScoreRow(ScoreUtils.lower(i + 5), scoreText));
+        }
+
+    }
+
+    // leeres Ergebnisfeld
+    private void updateScore() {
+        tableView.getItems().clear();
+        for (int i = 0; i < MAX_SCORE_NAME; i++) {
+            tableView.getItems().add(new ScoreRow(ScoreUtils.lower(i + 1), "", ""
+            ));
+        }
+    }
+
+    public void diceClick(@NotNull MouseEvent mouseEvent) {
+        if (this.gfx.rou.throwsLeft == 3 || this.gfx.rou.throwsLeft == 0) {
+            //debuggen
+            System.out.println("Es wurde noch nicht gewürfelt.");
+            setHelpPanel("Du musst zuerst Würfeln. Drück dafür 'Würfeln' ");
+            return;
+        }
+
+        int id = 0;
+        ImageView diceImg;
+        switch (mouseEvent.getPickResult().getIntersectedNode().getId()) {
```

Comment:
Auch diese könnte vereinfacht werden, wenn dice in einem Array wäre...

Reply:
array folgt noch in naher Zukunft...